pulpcore/tasking/tasks.py

```diff
@@ -363,6 +355,13 @@ def are_resources_available(colliding_resources, task: Task) -> bool:
     return not colliding_resources or not colliding_resources_taken
 
 
+def called_from_content_app() -> bool:
+    current_app = AppStatus.objects.current()
+    if current_app and current_app.type == "content":
+        return True
+    return False
```

Comment:
Nice!!!

Why not just return the expression you are branching on?
```suggestion
def called_from_content_app() -> bool:
    current_app = AppStatus.objects.current()
    return current_app is not None and current_app.type == "content"
```